README.md

```diff
@@ -1,22 +1,45 @@
-# dangermattic
-Shared collection of Danger plugins
+# Dangermattic
+`Dangermattic` builds on [Danger on Ruby](https://danger.systems/ruby/) and is essentially a collection of Danger plugins. Its goal is to provide customisable checks and common utilities to help perform checks on Pull Requests, from simple routine validations to more sophisticated ones.
 
 ## Installation
 
-Add to your `Gemfile`
+Add to your project's `Gemfile`
 ```
-gem 'dangermattic'
+gem 'danger-dangermattic', git: 'https://github.com/Automattic/dangermattic'
 ```
 
-## Usage
+## Example of available plugins and their usage
 
-    Methods and attributes from this plugin are available in
-    your `Dangerfile` under the `dangermattic` namespace.
+Once the main Gem is installed, all Dangermattic plugins are available in your `Dangerfile` under their corresponding namespace. A few examples:
+
+- `manifest_pr_checker` - Plugin to check if changes on a manifest file (i.e. `Gemfile`, `Podfile`) has a corresponding change in a lock file (i.e. `Gemfile.lock`, `Podfile.lock`)
+```ruby
+# Reports a warning if the Gemfile was changed but the Gemfile.lock wasn't
+manifest_pr_checker.check_gemfile_lock_updated
+```
+- `milestone_checker` - Plugin for performing checks on a milestone associated with a pull request
+```ruby
+# Checks if the pull request's milestone is due in 3 days or less, reporting a warning if that's the case
+milestone_checker.check_milestone_due_date(days_before_due: 3)
+```
+- `pr_size_checker` - Plugin to check the size of a Pull Request content and text body
+```ruby
+# Reports a warning if a pull request diff size is greater than 300
+pr_size_checker.check_diff_size(max_size: 300)
+```
+- `view_changes_need_screenshots` - Detects view changes in a PR and reports a warning if there are no attached screenshots
+```ruby
+# Reports a warning if a pull request changing views doesn't have a screenshot
+view_changes_need_screenshots.view_changes_need_screenshots
+```
```

Comment:
Having those all listed here kinda highlights the current discrepancy on their naming (3 of them ending in `_checker`, last one not), so I guess it's a reminder to standardize our naming convention of those plugins at some point in a future PR 🙂

Reply:
Nitpick/Tip: indenting the code blocks by 4 spaces will allow them to be considered as part of the bullet point that they're under (i.e. within the HTML's `<li>…</li>`) and making this a proper list of 4 bullets— instead of this being considered as 4 separate lists each with one item, and separated by a code block in between each like currently.

```suggestion
- `manifest_pr_checker` - Plugin to check if changes on a manifest file (i.e. `Gemfile`, `Podfile`) has a corresponding change in a lock file (i.e. `Gemfile.lock`, `Podfile.lock`)
    ```ruby
    # Reports a warning if the Gemfile was changed but the Gemfile.lock wasn't
    manifest_pr_checker.check_gemfile_lock_updated
    ```
- `milestone_checker` - Plugin for performing checks on a milestone associated with a pull request
    ```ruby
    # Checks if the pull request's milestone is due in 3 days or less, reporting a warning if that's the case
    milestone_checker.check_milestone_due_date(days_before_due: 3)
    ```
- `pr_size_checker` - Plugin to check the size of a Pull Request content and text body
    ```ruby
    # Reports a warning if a pull request diff size is greater than 300
    pr_size_checker.check_diff_size(max_size: 300)
    ```
- `view_changes_need_screenshots` - Detects view changes in a PR and reports a warning if there are no attached screenshots
    ```ruby
    # Reports a warning if a pull request changing views doesn't have a screenshot
    view_changes_need_screenshots.view_changes_need_screenshots
    ```
```